community/minutes/2024-11-27.md

```diff
@@ -0,0 +1,59 @@
+---
+tags: [meeting-notes]
+title: '2024-11-27'
+---
+# conda-forge core meeting 2024-11-27
+
+Add new agenda items under the `Your __new__() agenda items` heading
+
+- [Zoom link](https://zoom.us/j/9138593505?pwd=SWh3dE1IK05LV01Qa0FJZ1ZpMzJLZz09)
+- [What time is the meeting in my time zone](https://dateful.com/convert/utc?t=2pm)
+- [Previous meetings](https://conda-forge.org/community/minutes/)
+
+## Attendees
+
+| Name                    | Initials | GitHub ID        | Affiliation                 |
+| ----------------------- | -------- | ---------------  | --------------------------- |
+|                         |          |                  |                             |
+| Filipe Fernandes        | FF       | ocefpaf          | conda-forge                 |
+| Uwe Korn                | UK       | xhochy           | conda-forge/QuantCo         |
+| Wolf Vollprecht         | WV       | wolfv            | prefix.dev                  |
+| Marius van Niekerk        | MvN         | mariusvniekerk | conda-forge/Voltron Data |
+| Matthew Becker          | MRB      | beckermr         | cf                          |
+
+X people total
+
+### Standing items
+
+- [ ]
+
+### From previous meeting(s)
+
+- [ ]
+
+### Active votes
+
+- [ ]
+
+### Your __new__() agenda items
+
+- [x] (MRB) packages.conda-forge.org as canonical repodata location?
+    - idea is to redirect this to anaconda.org for now
+    - that could change in the future
+- [x] (MRB) `noarch: python` w/ `python_min` migration started
+    - this is a big one ~13k feedstocks
+    - doing one per bot run for now to fix errors
+- [x] (WV) multi-output cache for v1 recipes
+    - [x] Already available with `--experimental`
+    - [x] Wrote up a (proto)-CEP https://github.com/conda/ceps/pull/102
+- [x] (UK) `-fno-omit-frame-pointer`
```

Comment:
What was the outcome of this discussion? I remember that @jjerphan and colleagues were interested in this. AFAIU it's an ABI break though, so we'd probably have to queue that up for whenever we rebuild the world?

Reply:
Is it actually an ABI break to activate frame pointers? That sounds unexpected to me: you can compile any code with or without frame pointers, and still link to the same existing third-party or system libs.

Reply:
@xhochy was going to make a PR and see what people said

Reply:
Yes, but could it in practice alter some calling conventions?

I am not knowledgeable or experienced enough to be able to tell; I think it might be worth contacting people who re-activated them on RPM-based or DEB-based distributions.

Reply:
> For such a change, I would be super prudent and assume that ABI-compatibility depends on platforms.

Actually, I'm confused about whether this would change the ABI. My understanding is that a specific register is always filled, independent of whether the function uses it or not. I don't think this should be a world-rebuilt. 

Let's move this discussion to the PR over here: https://github.com/conda-forge/clang-compiler-activation-feedstock/pull/151